spec/controllers/devise/two_factor_authentication_controller_spec.rb

```diff
@@ -148,15 +148,6 @@ def index
           expect(flash[:success]).to eq nil
         end
       end
-
-      context 'multiple requests' do
-        it 'notifies the user of OTP transmission via flash message' do
```

Comment:
did we mean to remove this functionality? If so, are we replacing it with another visual indicator that a code was "re-sent" ? If not, why not? If so, test new functionality?